src/main/java/core/basesyntax/service/impl/TransactionHandlerImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.TransactionHandler;
+import core.basesyntax.strategy.OperationStrategy;
+import java.util.List;
+
+public class TransactionHandlerImpl implements TransactionHandler {
```

Comment:
this is more of a central service that contains the main logic, so I think this name fits more, considering that we have `OperationHandlers` with method `handle()` which can confuse us
```suggestion
public class FruitShopServiceImpl implements FruitShopService {
```